glean-core/ffi/examples/glean.h

```diff
@@ -323,6 +332,11 @@ uint8_t glean_timespan_test_has_value(uint64_t glean_handle,
                                       uint64_t metric_id,
                                       FfiStr storage_name);
 
+void glean_timing_distribution_accumulate_samples(uint64_t glean_handle,
+                                                  uint64_t metric_id,
+                                                  RawInt64Array raw_samples,
+                                                  int32_t num_samples);
+
```

Comment:
Is this change related to this PR?

Reply:
It's not directly, but I regenerated via cbindgen and this was missing apparently.

Reply:
Ok.  :+1: from me then.